src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/sessions/UpdateSessionTemplateDto.kt

```diff
@@ -28,13 +28,16 @@ data class UpdateSessionTemplateDto(
   @field:Valid
   val sessionCapacity: SessionCapacityDto?,
 
-  @Schema(description = "list of group references for permitted session location groups", required = false)
-  val locationGroupReferences: List<String>? = null,
-
   @Schema(description = "number of weeks until the weekly day is repeated", example = "1", required = true)
   @field:Min(1)
   val weeklyFrequency: Int?,
 
+  @Schema(description = "is prison active", example = "true", required = false)
+  val active: Boolean? = null,
```

Comment:
do we need this here, maybe?

Reply:
True, not needed. Removed from both create and update as suggested.